### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

**Comment by mookid** [comment]
HI @brotzeit,

I will take a look at it this weekend. Thanks.

**Comment by brotzeit** [comment]
I'm not sure what else would make sense to add. Do you want installation instructions ?

File: README.md [hunk 1]
```diff
@@ -1,96 +1,31 @@
+[![MELPA](https://melpa.org/packages/rust-mode-badge.svg)](https://melpa.org/#/rust-mode)
+
 `rust-mode`: A major Emacs mode for editing Rust source code
 ============================================================
 
-`rust-mode` makes editing [Rust](http://rust-lang.org) code with Emacs
-enjoyable.  `rust-mode` requires Emacs 24 or later.
```
Comment (mookid) [inline README.md]: I would keep emacs 24 as the minimally supported version

File: README.md [hunk 2]
```diff
@@ -1,96 +1,31 @@
...
-
-## Installation
-
-### Manual Installation
-
-To install manually, check out this repository and add this to your
-`.emacs` file:
-
-```lisp
```
Comment (mookid) [inline README.md]: I think that the install instructions are useful. Note that use-package does come as a built-in only for recent versions.

**Comment by brotzeit** [comment]
> I am not sure to understand the need for the complete rewrite of the README.

I noticed that some comments aren't up to date and others don't seem very useful so I decided to remove it and see what you are missing.

File: README.md (hunk 1 above)
Comment (brotzeit) [reply README.md]: Somebody mentioned there was a pull request lately that broke emacs24 compatibility.

Comment (mookid) [reply README.md]: @brotzeit this has been resolved and the code is still emacs24 compatible. Version 24 is still tested in CI.

**Comment by mookid** [comment]
OK to merge once the minimal required version (24 !) is back.

File: README.md [hunk 3]
```diff
@@ -1,96 +1,67 @@
...
 
-### `package.el` installation via MELPA
+# Installation via MELPA
 
-It can be more convenient to use Emacs's package manager to handle
-installation for you if you use many elisp libraries. If you have
-`package.el` but haven't added MELPA, the community
-package source, yet, add this to `~/.emacs.d/init.el`:
+`rust-mode` makes editing [Rust](http://rust-lang.org) code with Emacs
+enjoyable. It requires Emacs 24 or later.
```
Comment (brotzeit) [inline README.md]: I already added it again.

File: README.md [hunk 4]
```diff
@@ -1,96 +1,67 @@
...
+`M-x package-install rust-mode`
 
-MELPA stable only updates when a new version tag is created.  There isn't a
-specified frequency for this project to release versions; if you are a MELPA
-stable user and feel that the last release tag is too far out of date, open a
-pull request that updates the version header in `rust-mode.el` and note that a
-new version tag should be added when the request is merged.
-See [Pull Request #178](https://github.com/rust-lang/rust-mode/pull/178) (the
-first such pull request)) for an example of this.
+And put this in your config to load rustic automatically:
```
Comment (mookid) [inline README.md]: rustic -> rust-mode

**Pull request merged**